### ISSUE TITLE
[Bug]: KeyError: 'nsfw'

### ISSUE THREAD

**Issue by irldoggo** [BlafKing/sd-civitai-browser-plus#248]
### Describe the bug.

Another breaking API update it seems.

### Steps to reproduce the problem.

1. Press search
2. Error.

### Expected behavior

1. Press search.
2. No error.

### System info

* Extension version: latest
* OS: Arch
* SD-WebUI version: Forge latest
* Python: 3.10.3

### Console logs

```Shell
File "/home/***/sd/stable-diffusion-webui-forge/extensions/sd-civitai-browser-plus/scripts/civitai_api.py", line 224, in model_list_html
    if item["modelVersions"][0]["images"][0]['nsfw'] not in ["None", "Soft"]:
KeyError: 'nsfw'
```

**Comment by spooknik** [comment]
Also noticed this happening. I assume it's due to the changes civitai made to their NSFW filters.

**Comment by Hobo-Siah** [comment]
> Also noticed this happening. I assume it's due to the changes civitai made to their NSFW filters.
yep

**Comment by BlafKing** [comment]
Thanks for reporting, I'll post a fix for this shortly after figuring out what the new API information corelates to exactly 👍🏻

**Comment by Terroosh** [comment]
I hope Civitai can settle on something stable soon. These constant api changes must be annoying to deal with.

**Comment by BlafKing** [comment]
I have published a hotfix for this issue, it should now be fixed!
Unfortunately it looks like NSFW filtering is currently a bit broken after the API update from CivitAI.

I have created a bug report about the issue on the CivitAI github, so let's hope it gets fixed soon.
I have also created a bug report on this repo about the issues [here](https://github.com/BlafKing/sd-civitai-browser-plus/issues/249)

Thanks again for reporting!
I'll mark this as resolved 👍🏻
